source/__tests__/integration/export-profile.test.js

```diff
@@ -59,7 +60,6 @@ describe('Profiles Export', () => {
       await page.waitFor(1000, {waitUntil: 'networkidle2'})
       const valid_url_class = await page.$('input[name="resourceURI"]', e => e.getAttribute('ng-valid-url'))
       expect(valid_url_class).toBeTruthy()
```

Comment:
I believe this returns a promise because it's an async function?

Reply:
i believe so (i think that's the only thing an async function can return).  and i also believe the promise will resolve to null (or undefined?) since there's no return value for the function.